cmd/openshift-install/main.go

```diff
@@ -54,25 +54,25 @@ func installerMain() {
 
 func newRootCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:               "openshift-install",
-		Short:             "Creates OpenShift clusters",
-		Long:              "",
-		PersistentPreRunE: runRootCmd,
-		SilenceErrors:     true,
-		SilenceUsage:      true,
+		Use:              "openshift-install",
+		Short:            "Creates OpenShift clusters",
+		Long:             "",
+		PersistentPreRun: runRootCmd,
+		SilenceErrors:    true,
+		SilenceUsage:     true,
 	}
 	cmd.PersistentFlags().StringVar(&rootOpts.dir, "dir", ".", "assets directory")
 	cmd.PersistentFlags().StringVar(&rootOpts.logLevel, "log-level", "info", "log level (e.g. \"debug | info | warn | error\")")
 	return cmd
 }
 
-func runRootCmd(cmd *cobra.Command, args []string) error {
+func runRootCmd(cmd *cobra.Command, args []string) {
```

Comment:
ques: wouldn't just performing the level parsing check before setting `logrus.SetOutput(ioutil.Discard)` provide resolution to problem
```
If we can't parse the value, wait until we've setup stderr logging
before complaining about it.  With this commit:

  $ openshift-install --log-level destroy create cluster
  FATAL invalid log-level: not a valid logrus Level: "destroy"
  $ echo $?
  1

while previously:

  $ openshift-install --log-level destroy create cluster
  $ echo $?
  1
```

Reply:
> ques: wouldn't just performing the level parsing check before setting `logrus.SetOutput(ioutil.Discard)` provide resolution to problem

Sort of.  It would land before our stderr hook and its `&logrus.TextFormatter` config, so the output looks like:

```console
$ openshift-install --log-level destroy create cluster
FATA[0000] invalid log-level: not a valid logrus Level: "destroy" 
```

My weak preference is for 772b5145e6 as it stands, but I can switch this around if you'd prefer the earlier level parser approach.